stl/inc/expected

```diff
@@ -621,39 +621,39 @@ public:
     // [expected.object.obs]
     _NODISCARD constexpr const _Ty* operator->() const noexcept {
 #if _CONTAINER_DEBUG_LEVEL > 0
-        _STL_VERIFY(_Has_value, "expected stores an error, not a value");
-#endif // _CONTAINER_DEBUG_LEVEL > 0
+        _STL_VERIFY(_Has_value, "operator->() called on a std::expected that stores an error, not a value");

```

Comment:
Minor suggestion: "contains" instead of "stores".  "Store" to me usually implies a memory write.  "Contains" is more familiar, especially since this is a container.

Reply:
That's reasonable, I'll make that change. `expected` is not formally a container, but it plays one on TV. :tv:

Reply:
Done, thanks!